src/triage/cli.py

```diff
@@ -16,9 +19,29 @@ def __init__(self, parser):
 
 
 @Triage.register
-class ConfigVersion(Command):
-    """Return the experiment config version compatible with this installation of Triage"""
-    def __call__(self, args):
+class Experiment(Command):
+    """Validate and run experiments, manage experiment database"""
+    @cmdmethod
+    def upgradedb(self, args):
+        """Upgrade triage results database"""
+        logging.basicConfig(level=logging.INFO)
```

Comment:
up to you, but do you want to just factor this configuration statement to the module level?

Reply:
:+1: